### PR TITLE
Serverless computing and FaaS

### PR DESCRIPTION
**Pull request by anjakammer**
Addresses issue #23 

This proposal includes the introduction of the serverless concept in Chapter 1 Fundamentals of Modern Infrastructures.
Related to this, the use of Functions as a Service technology for operating services is introduced in Chapter 4 Patterns.

Note: Serverless Computing and FaaS are not used as synonyms.

### CODE REVIEW

File: docs/04-Patterns/01-duration-terms.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@
 |===
 
 === Begriffe und Konzepte
-Resilienz Muster, Container Application Design, Cloud Native Architekturen, Container Pattern, Service Mesh
+Resilienz Muster, Container Application Design, Cloud Native Architekturen, Functions as a Service (FaaS), Container Pattern, Service Mesh
```
Comment (sw4ma-marhai) [inline docs/04-Patterns/01-duration-terms.adoc]: Maybe it is better to change the order like Container Pattern, Functions as a Service (FaaS), cause FaaS is based on the Container Pattern

File: docs/04-Patterns/02-learning-goals.adoc [hunk 2]
```diff
@@ -19,7 +19,19 @@ Softwarearchitekt:innen kennen Methoden, um technische von fachlichen Aufgaben d
...
 
 [[LZ-4-2]]
-==== LZ 4-2: Passende Resilienzmuster zur Erhöhung von Fehlertoleranz auswählen
+==== LZ 4-2: Passende Technologien für den Betrieb von Modulen auswählen
+
+Softwarearchitekt:innen können geeignete Technologien zum Betrieb von Modulen eines verteilten Systems auswählen, z.B. mit Functions as a Service (FaaS) und Container Orchestrierung.
+
+Darüber hinaus können Sie die Anwendung dieser Technologien Anforderungsgetrieben anwenden. Dabei gibt es verschiedene Aspekte zu beachten wie:
+
+* Skalierungsanforderungen und Ausführungszeit
```
Comment (sw4ma-marhai) [inline docs/04-Patterns/02-learning-goals.adoc]: Include Ausführungszeit also the time span to start up the function?

Comment (anjakammer) [reply docs/04-Patterns/02-learning-goals.adoc]: I included the time to startup in commit: [f9adf03](https://github.com/isaqb-org/curriculum-cloudinfra/pull/25/commits/f9adf03586e9f58e7a56ebad9b8363986ed59ee0)

File: docs/04-Patterns/01-duration-terms.adoc (hunk 1 above)
Comment (anjakammer) [reply docs/04-Patterns/01-duration-terms.adoc]: I changed the order in this commit: [4e9f8b5](https://github.com/isaqb-org/curriculum-cloudinfra/pull/25/commits/4e9f8b5557e3fbfa8e6f361a20a0827244468043)

**Review by sw4ma-marhai — APPROVED** [review]

**Comment by anjakammer** [comment]
closes #23

**Pull request merged**